src/vs/workbench/contrib/workspace/browser/workspaceTrustEditor.css

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-icon-label.file-icon.workspacetrusteditor-name-file-icon.ext-file-icon.tab-label::before {
+	font-family: 'codicon';
+	content: '\eb53';
```

Comment:
Isn't there a codicon class you are supposed to use instead of this?

Reply:
editor tabs are controlled by the file icon theme and not codicons... cc @misolori

Reply:
We've had requests to allow codicons in the editor tabs cc @aeschli